### PR TITLE
Minor changes in PHP source code

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/controllers/AdminloginController.php [hunk 1]
```diff
@@ -3,6 +3,16 @@
 
 class AdminloginController extends Base
 {
+    private static function getHashedPassword($password)
+    {
+        return sha1($password);
+    }
+
+    private static function isCorrectPassword($exceptedPassword, $givenPassword)
```
Comment (ghost) [inline app/controllers/AdminloginController.php:11]: Would be useful to make https://github.com/movim/movim/pull/469 more readable.

**Comment by Ppjet6** [comment]
Hi there!

Your changes look helpful indeed, thanks! I was just wondering if you could maybe try to split them into meaningful commits, instead of putting everything in the same bag.
Almost none of these changes look semantically related to me, maybe that could be a nice addition to your next PR :)

**Comment by edhelas** [comment]
Indeed, there's some things in those PR that I agree and others that would need discussion. So I think it's better to split all those changes in smaller uniform chunks, if possible :)

**Comment by ghost** [comment]
When I do minor changes, I prefer to do one commit instead of even smaller ones. But as you prefer the contrary, I will split.

**Pull request closed without merging**